samples/training/s3_sample_data.py

```diff
@@ -0,0 +1,21 @@
+import urllib.request
+from urllib.parse import urlparse
+
+###################################################################
+# This is the only thing that you need to change to run this code
+# Give the name of your S3 bucket
+bucket = "YOUR-BUCKET-NAME"
+###################################################################
+
+
+
+## Download dataset from pinned commit
+url = "https://github.com/aws/amazon-sagemaker-examples/raw/af6667bd0be3c9cdec23fecda7f0be6d0e3fa3ea/sagemaker-debugger/xgboost_realtime_analysis/data_utils.py"
+urllib.request.urlretrieve(url, "data_utils.py")
+
+from data_utils import load_mnist, upload_to_s3
+
+prefix = "sagemaker/xgboost"
+train_file, validation_file = load_mnist()
+upload_to_s3(train_file, bucket, f"{prefix}/train/mnist.train.libsvm")
+upload_to_s3(validation_file, bucket, f"{prefix}/validation/mnist.validation.libsvm")
```

Comment:
can you run `black` on this file  if you have not already ? I think black adds a newline at the EOF.